packages/react/package.json

```diff
@@ -64,15 +64,15 @@
     "@babel/preset-env": "^7.4.3",
     "@babel/preset-react": "^7.0.0",
     "@carbon/test-utils": "^10.6.0-rc.0",
-    "@storybook/addon-a11y": "^5.2.1",
-    "@storybook/addon-actions": "^5.2.1",
-    "@storybook/addon-info": "^5.2.1",
-    "@storybook/addon-knobs": "^5.2.1",
-    "@storybook/addon-links": "^5.2.1",
-    "@storybook/addon-storysource": "^5.2.1",
-    "@storybook/addons": "^5.2.1",
-    "@storybook/react": "^5.2.1",
-    "@storybook/source-loader": "^5.2.1",
+    "@storybook/addon-a11y": "^5.2.3",
```

Comment:
For storybook updates should we pull these into a separate PR?

Reply:
The only reason that I included the updates was due to a single bug that was fixed in 5.2.2, PR here: https://github.com/storybookjs/storybook/pull/8006.
Essentially, I wanted a "blank" default option in the knobs to pass undefined but I had to use the string "undefined" and parse it due to the above.

Reply:
resolved via 573727e